lib/pages/forward/recent_chat_title.dart

```diff
@@ -0,0 +1,49 @@
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:fluffychat/widgets/twake_components/twake_icon_button.dart';
+import 'package:flutter/material.dart';
+
+class RecentChatsTitle extends StatelessWidget {
+  const RecentChatsTitle({
+    super.key,
+    required this.isShowRecentlyChatsNotifier,
+    required this.toggleRecentChat,
+  });
+
+  final ValueNotifier<bool> isShowRecentlyChatsNotifier;
+
+  final void Function() toggleRecentChat;
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: [
+        Text(
+          L10n.of(context)!.recentlyChats,
+          style: Theme.of(context).textTheme.titleLarge,
+        ),
+        Expanded(
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.end,
+            children: [
+              TwakeIconButton(
+                paddingAll: 6.0,
+                buttonDecoration: BoxDecoration(
+                  color:
+                      Theme.of(context).colorScheme.onSurface.withOpacity(0.12),
+                  shape: BoxShape.circle,
+                ),
+                icon: isShowRecentlyChatsNotifier.value
```

Comment:
pass bool instead ValueNotifier<bool>
or you should add ValueListenableBuilder inside RecentChatsTitle